stats/opentelemetry/opentelemetry.go

```diff
@@ -248,37 +249,40 @@ type serverMetrics struct {
 }
 
 func createInt64Counter(setOfMetrics map[Metric]bool, metricName Metric, meter metric.Meter, options ...metric.Int64CounterOption) metric.Int64Counter {
+	nop := noop.Int64Counter{}
```

Comment:
Super minor thing, but it's a little unusual to have this variable scoped here and instantiated even when you don't end up using it.  Just `return noop.Int64Counter{}`?  Or if you wanted you could make a global nop counter for each type and ref them in the return instead.

Nice that they already had a nop implementation though!

Reply:
Alright I'll return inline. I was wondering the nicest way to do this too. I'd rather not make these globals to not populate namespace where only time you need it is these create functions. Thanks for the passes, this looks much better than when it started!